student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] To avoid errors you need to use the appropriate argument definition for `parent_permission` here.

  ```
  super(age, name, parent_permission: parent_permission)
  ```